### DISCUSSION TITLE
【Via浏览器】上使用该脚本设置界面会显示不全

### DISCUSSION THREAD

**Discussion by DreamProStudio** [F9y4ng/GreasyFork-Scripts#406] [问答专区(Question / Answer)]
如题见图🤔
![Screenshot_2024-12-06-01-58-47-688_mark via](https://github.com/user-attachments/assets/354e4507-eab2-4770-adb1-19b892825e81)
![Screenshot_2024-12-06-01-59-24-678_mark via](https://github.com/user-attachments/assets/867c9fb4-722d-48e8-8811-26592a62e1eb)

**Comment by F9y4ng** [comment]
脚本仅适配桌面浏览器。

**Reply by DreamProStudio** [reply]
好的(•̤̀ᵕ•̤́๑)ᵒᵏᵎᵎᵎᵎ
